packages/camera/camera_android_camerax/lib/src/camera_control.dart

```diff
@@ -69,6 +78,18 @@ class _CameraControlHostApiImpl extends CameraControlHostApi {
           .add(e.message ?? 'The camera was unable to change torch modes.');
     }
   }
+
+  /// Sets zoom of specified [CameraControl] instance by ratio.
+  Future<void> setZoomRatioFromInstance(
+      CameraControl instance, double ratio) async {
+    final int identifier = instanceManager.getIdentifier(instance)!;
+    try {
+      await setZoomRatio(identifier, ratio);
+    } on PlatformException catch (e) {
+      SystemServices.cameraErrorStreamController.add(e.message ??
```

Comment:
This is true, but this follows suit from the other camera platform implementations. I would argue that it's okay that we don't include it because we should only be catching camera errors like this when the camera was unable to complete the operation and not due to a code error. If you feel strongly though, I don't think it would hurt to include!